src/crimson/CMakeLists.txt

```diff
@@ -1,6 +1,13 @@
 add_library(crimson::cflags INTERFACE IMPORTED)
+set(crimson_cflag_definitions "WITH_SEASTAR=1")
+# disable concepts to address https://github.com/boostorg/asio/issues/312
+if((CMAKE_CXX_COMPILER_ID STREQUAL GNU AND CMAKE_CXX_COMPILER_VERSION VERSION_GREATER_EQUAL 10) OR
```

Comment:
for those who are interested in a minimal reproducer, see https://www.godbolt.org/z/6igCM2